build/tools.go

```diff
@@ -17,10 +17,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// This package imports things required by build scripts, to force `go mod` to see them as dependencies
+// This package imports things required by build scripts and test packages of submodules, to force `go mod` to see them as dependencies
 package tools
 
 import (
+	// build script dependencies
+	_ "github.com/go-bindata/go-bindata/go-bindata"
```

Comment:
this shouldn't be needed any more

Reply:
It was a leftover from the previous PR, I removed the vendor folder and all the references inside build/tools + go.mod and go.sum.